copulas/bivariate/gumbel.py

```diff
@@ -25,9 +25,11 @@ def probability_density(self, X):
 
             \begin{align}
                 c(U,V)
-                    &= \frac{\partial^2 C(u,v)}{\partial v \partial u} \\
-                    &= \frac{C(u,v)}{uv} \frac{((-\ln u)^{\theta} + (-\ln v)^{\theta})^{\frac{2}
-                {\theta} - 2 }}{(\ln u \ln v)^{1 - \theta}} ( 1 + (\theta-1) \big((-\ln u)^\theta
+                    &= \frac{\partial^2 C(u,v)}{\partial v \partial u}
+                    &= \frac{C(u,v)}{uv} \frac{((-\ln u)^{\theta}  # noqa: JS101
```

Comment:
These are multiline fractions. Since I couldn't find a way to write them in one line, I'm using `noqa`.